src/main.ts

```diff
@@ -17,33 +15,31 @@ import {
     USE_REQUEST_ANIMATION_FRAME,
     DEBUG,
 } from './constants';
-import { PopularResponseType, RowImageType, RowObjectType } from './types';
-
+import {
+    getRowsData,
+    loadAndRenderImages,
+} from './api';
+import { RowObjectType } from './types';
 
 // TODO move to helper functions module
 const easeOutQuint = (delta: number) => {
     return 1 - Math.pow(1 - delta, 5);
 }
 
 // BUG fast up and down causes glitch - prevDirection and a timer hack used until this is fixed properly
-// TODO move API stuff to seperate module
+// TODO add carousel row titles - show highlighted carousel row title to the right that fades out over time
```

Comment:
It would be nice to have these things as issues here on the repo =)

Reply:
Great idea. Will do so.